.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,21 @@
+# https://pre-commit.com
+# This GitHub Action assumes that the repo contains a valid .pre-commit-config.yaml file.
+name: pre-commit
+on:
+  pull_request:
+  push:
+    branches: [master]
```

Comment:
Should this be [main], as that is the default branch of TinyScaler?

Reply:
Sorry, yes, I will update